### PR TITLE
Wl 141 tezos design tool

### PR DESCRIPTION
**Pull request by dev-tronghieu**

### CODE REVIEW

**Comment by tanlethanh** [comment]
I have some fixes to make the design tool can fetch metadata of the Tezos network.

**Comment by fluctlight-kayaba** [comment]
need clean-up commit history of this PR before reviews.

**Comment by tanlethanh** [comment]
@fluctlight-kayaba I have just cleaned the commit history, pls review this PR.

File: apps/docs/package.json
Comment (fluctlight-kayaba) [inline apps/docs/package.json:1]: I wonder if we really need this file?

**Review by fluctlight-kayaba — APPROVED** [review]

**Pull request merged**